services/collaboration/pkg/connector/fileconnector.go

```diff
@@ -533,6 +534,7 @@ func (f *FileConnector) CheckFileInfo(ctx context.Context) (FileInfo, error) {
 		fileInfo.DisableExport = true
 		fileInfo.DisableCopy = true
 		fileInfo.DisablePrint = true
+		fileInfo.WatermarkText = f.ensureWatermarkText(wopiContext.User)
```

Comment:
Maybe we should include a comment saying that this will affect only to collabora. The rest of the wopi apps don't use that property.

Reply:
It is already documented in the `FileInfo struct` definition https://github.com/owncloud/ocis/blob/master/services/collaboration/pkg/connector/fileinfo.go#L172-L231
Do you think we should duplicate that?

Reply:
I think so. I mean, regardless of the actual wopi app being used, the property will be set.

If I use onlyoffice, for example, and I go through that code but I don't see the watermark showing, I'll have to check what's going on and why it doesn't show.

We probably need to refactor that method at some point to simplify the code, but for now just a `Collabora only` comment might be enough as a reminder.